src/Main.java

```diff
@@ -1,5 +1,77 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Привет мир!");
+
+        char neZnau = 569 ;
+        int a = -159;
+        byte b = 67;
+        short c = 27897 ;
+        long d = 987678965549L ;
+        float e = 27.12f ;
+        double camelCase = 2.786 ;
+        System.out.println(neZnau);
+        System.out.println(a);
+        System.out.println(b);
+        System.out.println(c);
+        System.out.println(d);
+        System.out.println(e);
+        System.out.println(camelCase);
+
+        byte teacherLP = 23 ;
+        byte teacherAC = 27 ;
+        byte teacherEA = 30 ;
+        short sheets = 480 ;
+        float gotIt =  sheets/(teacherLP + teacherAC + teacherEA);
+        System.out.println("На каждого ученика рассчитано "+gotIt+" листов бумаги");
+
+        byte performanceOfTheBottleMakingMachineTranslation = 16;
+        int bottleMachine = (byte) (performanceOfTheBottleMakingMachineTranslation/2);
+        int in20minutes =  20 * bottleMachine;
+        int perDay =  24 * 60 * bottleMachine;
+        int in3days =  3 * perDay;
+        int for1month = 10 * in3days;
+        System.out.println ("За "+20+" минут машина произвела "+in20minutes+" штук бутылок") ;
+        System.out.println ("За сутки машина произвела "+perDay+" штук бутылок") ;
+        System.out.println ("За "+3+" дня машина произвела "+in3days+" штук бутылок") ;
+        System.out.println ("За месяц машина произвела "+for1month+" штук бутылок") ;
+
+        byte pot = 120;
+        byte ForOneClass = 6;
```

Comment:
Названия переменных называются с маленькой буквы
```suggestion
        byte forOneClass = 6;
```